test/system/075-exec.bats

```diff
@@ -166,6 +166,52 @@ load helpers
     run_podman rm -f -t0 $cid
 }
 
+@test "podman exec --tty" {
+    # Outer loops: different variations on the RUN container
+    for run_opt_t in "" "-t"; do
+        for run_term_env in "" "explicit_RUN_term"; do
+            local run_opt_env=
+            if [[ -n "$run_term_env" ]]; then
+                run_opt_env="--env=TERM=$run_term_env"
+            fi
+            run_podman run -d $run_opt_t $run_opt_env --name test $IMAGE top
+
+            # Inner loops: different variations on EXEC
+            for exec_opt_t in "" "-t"; do
+                for exec_term_env in "" "explicit_EXEC_term"; do
+                    # What to expect.
+                    local expected=
+                    # "exec -t" defaults to the process xterm
+                    if [[ -n "$exec_opt_t" ]]; then
+                        expected="xterm"
+                    fi
+                    # ...unless "run -t", which forces the run-process xterm
```

Comment:
here too

Reply:
Actually, this block is now absurd. It should be collapsed, both `if`s should become:
```bash
    # if -t is set anywhere, either run or exec, go with xterm
    if [[ -n "$run_opt_t$exec_opt_t" ]]; then
```